pwndbg/commands/__init__.py

```diff
@@ -708,6 +708,7 @@ def load_commands() -> None:
         import pwndbg.commands.dumpargs
         import pwndbg.commands.elf
         import pwndbg.commands.flags
+        import pwndbg.commands.gdt
```

Comment:
Just a small question. You don't seem to use `gdb` or `gdblib` at all in the command. Couldn't this import go outside the `if` for GDB-only commands?

Reply:
I believe it should. I just didn't know where to put it. Please do it when u port some other commands to aglib